docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -40,7 +42,12 @@ Set the systemd restart policy.  The restart-policy must be one of: "no", "on-su
 
 ## Examples
 
-Create and print a systemd unit file for a container running nginx with an *always* restart policy and 1-second timeout to stdout.
+### Generate unit files
+
+#### Generate and print a systemd unit file for a container
+
+Generates  systemd unit file for container running nginx with an *always* restart policy and 1-second timeout to stdout.
```

Comment:
```suggestion
Generate systemd unit file for container running nginx with an *always* restart policy and a 1-second timeout to stdout.
```

Reply:
And further add to @vrotheberg: "Generate a systemd unit file for a container ..."